unit-tests/live/frames/test-fps-permutations.py

```diff
@@ -0,0 +1,304 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:donotrun:!nightly
+# test:timeout 750
+# timeout - on the worst case, we're testing on D585S, which have 8 sensors, so:
```

Comment:
The term should be 8 streams (not sensors).
In realsense a sensor can group multiple streams of data.
"Stereo Module" sensor groups IR1+IR2+depth streams. "Motion Module" sensor groups gyro+accel.
The grouped streams usually have some physical relation between them, but not necessarily so, and are camera model specific.